.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "wat-storybook"
```

Comment:
The name can be changed to something generic since it will also have different deployments apart from storybook

Reply:
Updated it to `web-app-template-1bd48`